app/controllers/logout_controller.rb

```diff
@@ -0,0 +1,8 @@
+class LogoutController < ApplicationController
```

Comment:
Shouldn't need this controller if we are u sing the logout method in auth0_controller